### ISSUE TITLE
Tutorial on best practices for training video loras (hunyuan example)

### ISSUE THREAD

**Issue by paul-lupu** [kohya-ss/musubi-tuner#105]
Heyo, 

Spoke with someone that did a lot of really really high quality loras and he was gracious enough to summarize our talk in this article: https://civitai.com/articles/11942/training-a-lora-the-right-way

The stuff he does is amazing with no hand issue, artifacting etc so definitely recommend giving it a read; 

@Sarania I know you mentioned you're also having issues. 

Also one thing i don't think he mentioned: if you want to stack loras, it's easier if they are dim 16 instead of 32.

**Comment by Sarania** [comment]
Hey I appreciate it. Most of that was already stuff I was doing, though I am training at a significantly higher resolution than is suggested there so I might try lowering it. I think my LR has been too high, and I've been experimenting with a lower LR but LoraPlus and that might have improved it. I'm not really having hand issues anymore(lowering LR helped a lot there, also sticking with shift/7 for video training) but issues in fine detail separation.

**Comment by Sarania** [comment]
@paul-lupu Hey I finally got around to training something new and yes indeed using a lower res seems to work just as well, so thanks for that! I also trained it at Rank 16 and got exceptional results. It's also good to see that most of my other practices, especially my captions, were in line with what the guide recommends. Some of my faces were out of frame, the others I blurred out with a simple python script that o1 wrote and mentioned "censored face" in the captions. This did good to produce a non face altering model! If I learn anything else useful, I'll post back!

**Comment by frosty3907** [comment]
what learning rate are you guys using? I'm trying lora+ now and in A/B testing it's proving inferior so far.

**Comment by Sarania** [comment]
For Hunyuan I've had the most luck with an LR of around 1.6e-5, loraplus_lr_ratio=4, network dim = 16, network alpha = 16. Adding a little network dropout (~5%) can help with stubborn targets as well. I haven't done any A/B with Hunyuan and loraplus so I'm interested in your results! I /have/ tested with Wan 2.1 and it's definitely beneficial there so I've just been using it since tbh.

Edit: Oh and for reference settings are very similar for Wan but I usually use LR of 2e-5. You definitely need to lower your base LR when using LoraPlus so you don't cook the B blocks compared to if not using it.
